.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
@samhill303 Do I need to update JDK?

Reply:
Possibly. This never seems to stop thrashing. check that it is set to use the bundled jdk
https://stackoverflow.com/a/40083824
if it is maybe it's still getting confused, consider all of your jdk instances